apps/xero/utils.py

```
@@ -65,25 +74,112 @@
+        )
+
+        for suppliers in supplier_generator:
+            supplier_attributes = []
 
-        for suppliers in suppliers_generator:
             for supplier in suppliers["Contacts"]:
-                merchant_names.append(supplier["Name"])
+                detail = {
+                    "email": supplier["EmailAddress"]
+                    if "EmailAddress" in supplier
+                    else None
```

Ruff (0.11.9)

Use `supplier.get("EmailAddress", None)` instead of an `if` block

Replace with `supplier.get("EmailAddress", None)`

(SIM401)